packages/vue-country-flag/src/CountryFlag.vue

```diff
@@ -65,12 +65,14 @@ export default {
 </script>
 <style scoped>
   .small-flag {
+    margin: -0.9em -1.2em -0.9em -1.2em;
     transform: scale(0.25);
     -ms-transform: scale(0.25); 
     -webkit-transform: scale(0.25);
     -moz-transform: scale(0.25);
   }
   .normal-flag {
+    margin: 0em -0.9em -0.6em -0.7em;
```

Comment:
I think it should be 
```css
margin: -10px -13px;
```
because your original image is 52x39, scale by 0.5 : 26x19.5, divided by 2 : 13x9.75